hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/MerkleTestBase.java

```diff
@@ -75,12 +84,18 @@ public class MerkleTestBase extends StateTestBase {
     public static final String SECOND_SERVICE = "Second-Service";
     public static final String UNKNOWN_SERVICE = "Bogus-Service";
 
-    /** A TEST ONLY {@link Codec} to be used with String data types */
+    /**
```

Comment:
Why did we go in and reformat all the comments?

Reply:
Accidental reformatting in Idea, I suppose. I'll roll it back.